song/app.js

```diff
@@ -31,6 +31,22 @@ app.get("/ping", function (req, res, next) {
   res.json({ message: "pong" });
 });
 
+app.post("/join", async (req, res, next) => {
+  const { id, name, email, profile_image, password } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO users(
+      id,
+      name,
+      email,
+      profile_image,
+      password
+    ) VALUES (?, ?, ?, ?, ?) `,
+    [id, name, email, profile_image, password]
+  );
```

Comment:
```suggestion
  await appDataSource.query(
    `INSERT INTO users(
      name,
      email,
      profile_image,
      password
    ) VALUES (?, ?, ?, ?) `,
    [name, email, profileImage, password]
  );
```
users 테이블을 DDL을 통하여 정의내릴 때, id(pk)에 auto increment 옵션을 주었기 때문에, insert into 문에서 값을 넣어주지 않아도 괜찮습니다.